front/src/Components/Events.js

```diff
@@ -10,6 +10,7 @@ import EventsTab from './EventsTab';
 import EventBox from './EventBox';
 import GithubIcon from '../icons/github.svg';
 import FacebookIcon from '../icons/facebook.svg';
+import SearchBox from './SearchBox';
```

Comment:
Module not found: Error: Can't resolve './SearchBox' in '/usr/local/go_appengine/gopath/src/github.com/enow-dev/enow/front/src/Components'